shell/lisa_shell

```diff
@@ -288,6 +289,22 @@ function _lisa-jupyter-start {
         return 1
     fi
 
+    # Check required port is available (e.g. Vagrant could using the same port)
+    netstat -nlt | grep $PORT &>/dev/null; RET=$?
+    if [[ $RET -eq 0 ]]; then
+        echo
+        echo "ERROR: required port [$PORT] already in use"
+        echo "       try again with a different PORT option"
+        echo
+	_lisa-jupyter-usage
+	return 1
+    fi
+
+    # Check for jupyter-lab
+    if [[ -f $LISA_VENV_PATH/bin/jupyter-lab ]]; then
```

Comment:
Ok, so in that case we just install jupyterlab as a dependency, and we use that command. It will be installed by lisa-install so there is no need to check for it (and if the user does not install the "notebook" extra dep because he did a manual install without lisa-install, that's not something we want to try to catch).

Reply:
That's what the new version does ;)